concepts/05 UI Components/Form/01 Getting Started with Form/20 Populate the Form Fields.md

```diff
@@ -0,0 +1,124 @@
+To create a data entry UI, assign an object to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. Each data field in this object produces a [simple item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/)&mdash;a label-editor pair. 
```

Comment:
"produces" Explain

Reply:
The Form creates a separate simple item for each field in the formData object

Reply:
I would use this instead of 'produces'